cmd/nvidia-dra-plugin/main.go

```diff
@@ -108,11 +108,11 @@ func newApp() *cli.App {
 			EnvVars:     []string{"CONTAINER_DRIVER_ROOT"},
 		},
 		&cli.StringFlag{
-			Name:        "nvidia-ctk-path",
-			Value:       "/usr/bin/nvidia-ctk",
-			Usage:       "the path to use for the nvidia-ctk in the generated CDI specification. Note that this represents the path on the host.",
-			Destination: &flags.nvidiaCTKPath,
-			EnvVars:     []string{"NVIDIA_CTK_PATH"},
+			Name:        "nvidia-cdi-hook-path",
+			Value:       "/usr/bin/nvidia-cdi-hook",
+			Usage:       "the path to use for the nvidia-cdi-hook in the generated CDI specification. Note that this represents the path on the host.",
+			Destination: &flags.nvidiaCDIHookPath,
+			EnvVars:     []string{"NVIDIA_CDI_HOOK_PATH"},
```

Comment:
Do we want to still support the `NVIDIA_CKT_PATH` envvar as a transitional option?
```suggestion
			EnvVars:     []string{"NVIDIA_CDI_HOOK_PATH", "NVIDIA_CTK_PATH"},
```

Reply:
Good advice.